halo2_proofs/Cargo.toml

```diff
@@ -30,6 +30,7 @@ harness = false
 [[bench]]
 name = "commit_zk"
 harness = false
+required-features = ["multicore"]
```

Comment:
Because only it imports `maybe_rayon::current_num_threads`, which doesn't exist when `maybe_rayon/thread` is disabled.

I've fixed it in https://github.com/privacy-scaling-explorations/halo2/pull/199/commits/b848963cf51aba8542091da8c4b2c5a573f442a2 so we could also run it with `--no-default-features`.